backend/src/main/kotlin/org/loculus/backend/service/submission/SubmissionDatabaseService.kt

```diff
@@ -198,37 +203,24 @@ open class SubmissionDatabaseService(
         log.info { "updating processed data" }
         val reader = BufferedReader(InputStreamReader(inputStream))
 
-        val statusByAccessionVersion = mutableMapOf<String, PreprocessingStatus>()
+        val processedAccessionVersions = mutableListOf<String>()
         reader.lineSequence().forEach { line ->
             val submittedProcessedData = try {
                 objectMapper.readValue<SubmittedProcessedData>(line)
             } catch (e: JacksonException) {
                 throw BadRequestException("Failed to deserialize NDJSON line: ${e.message}", e)
             }
 
-            val newStatus = insertProcessedDataWithStatus(submittedProcessedData, organism, pipelineVersion)
-
-            statusByAccessionVersion[submittedProcessedData.displayAccessionVersion()] = newStatus
+            insertProcessedDataWithStatus(submittedProcessedData, organism, pipelineVersion)
```

Comment:
```suggestion
            insertProcessedDataWithStatus(submittedProcessedData, organism, pipelineVersion)
```

Does the "WithStatus" still make sense? we no longer do anything with the return value